index.html

```diff
@@ -49,7 +31,7 @@
       <div class="logo_group">
         <a href="#"
           ><p class="logo-text">Linked</p>
-          <img src="/ProjectUI_Navbar/img/logo/linkedin_128px.png" />
+          <img src="img/logo/linkedin_128px.png" />
```

Comment:
You were using absolute paths to access assets. I've changed it to relative paths. Avoid using absolute paths. Always prefer relative paths.